src/modules/recipes/CreateNewRecipe/index.jsx

```diff
@@ -0,0 +1,273 @@
+              },
+            })}
+          />
+          {errors.strMeal && <p>{errors.strMeal.message}</p>}
+
+          <label htmlFor="strCategory">Select Category</label>
+          <select id="strCategory" {...register("strCategory")}>
+            {categories.map((category) => (
+              <option key={category} value={category}>
+                {category}
+              </option>
+            ))}
+            required
+          </select>
+
+          <label htmlFor="strArea">Select Area</label>
+          <select id="strArea" {...register("strArea")}>
+            {areas.map((area) => (
+              <option key={area} value={area}>
+                {area}
+              </option>
+            ))}
+            required
+          </select>
+
+          <RecipeForm onSubmit={handleRecipeSubmit} />
+
+          <label htmlFor="strInstructions">Cooking Instructions</label>
+          <textarea
+            className="instructions"
+            id="strInstructions"
+            {...register("strInstructions")}
+            required
+          />
+          <label htmlFor="strMealThumb">Meal Photo</label>
+          <input
+            type="file"
+            style={{ width: "100% " }}
+            id="strMealThumb"
+            {...register("strMealThumb")}
+            required
+          />
+
+          <label htmlFor="strYoutube">You Tube</label>
+          <input
+            type="text"
+            style={{ width: "100% " }}
+            id="strYoutube"
+            {...register("strYoutube", {
+              pattern: {
+                value: /^(https?:\/\/)?(www\.)?(youtube\.com|youtu\.?be)\/.+$/,
+                message: "Geçerli bir YouTube bağlantisi girin.",
+              },
+            })}
+          />
+
+          <label htmlFor="strSource">Source</label>
+          <input
+            type="text"
+            style={{ width: "100% " }}
+            id="strSource"
+            {...register("strSource", {
+              value: "#",
+            })}
+          />
+
+          <label htmlFor="strImageSource">Image Source</label>
+          <input
+            type="text"
+            style={{ width: "100% " }}
+            id="strImageSource"
+            {...register("strSource", {
+              value: "#",
+            })}
+          />
+
+          <div style={{ display: "flex", alignItems: "center" }}>
+            <label
+              htmlFor="strCreativeCommonsConfirmed"
+              style={{ marginRight: "5px" }}
+            >
+              Has A Creative Common:
+              <input
+                type="checkbox"
+                style={{ width: "30px", marginLeft: "10px" }}
+                id="strCreativeCommonsConfirmed"
+                {...register("strCreativeCommonsConfirmed")}
+              />
+            </label>
+          </div>
+          <button type="submit">Save Recipe</button>
+          {successMessage && <p>{successMessage}</p>}
```

Comment:
It's really hard to see the message at the bottom of the page, try to make it more visible so it's very clear to the user that the form is not submitted.